ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/TokenAuthentication/README.md

```diff
@@ -25,7 +25,11 @@ Once you launch the app, you will be challenged for an ArcGIS Online login to vi
 
 ## About the data
 
-The Traffic web map uses public layers as well as the world traffic (premium content) layer. The world traffic service presents historical and near real-time traffic information for different regions in the world. The data is updated every 5 minutes. This map service requires an ArcGIS Online organizational subscription.
+The [Traffic web map](https://arcgisruntime.maps.arcgis.com/home/item.html?id=e5039444ef3c48b8a8fdc9227f9be7c1) uses public layers as well as the world traffic (premium content) layer. The world traffic service presents historical and near real-time traffic information for different regions in the world. The data is updated every 5 minutes. This map service requires an ArcGIS Online organizational subscription.
```

Comment:
this sample differs from the other implementations - the new implementations use the traffic service but the Qt sample still uses the ArcGIS Server secured sampleserver6 layer. Therefore, this (and some of the previous info about the traffic map) should be removed until we refactor the sample.

Reply:
Sounds good. I will reverse the commit and remove the traffic map info.